webapi/_test_data/ontologies/beol-onto.ttl

```diff
@@ -1453,6 +1453,43 @@
                    rdfs:comment """Dieser Brief ist auch vorhanden in"""@de ,
                               """Letter is also available at"""@en .
 
+### ###########################################
+### beol:entryCommentOf
+
+:entryCommentOf rdf:type owl:ObjectProperty ;
+
+                rdfs:subPropertyOf knora-base:hasLinkTo ;
+
+                rdfs:comment "Kommentar zu"@de ,
+                             "Comment of"@en ;
+
+                rdfs:label  "Kommentar von"@de ,
```

Comment:
shouldn't this also be "Kommentar zu"?

Reply:
done in https://github.com/dhlab-basel/Knora/pull/1281/commits/37e51f9e5c622ab25aa90032aabd6c1353549970